### ISSUE TITLE
Some SA2s turn black/transparent at particular zoom levels

### ISSUE THREAD

**Issue by justinanderson** [CxSci/SA-dashboard#43]
At certain zoom levels, the map appears to stop drawing parts of some of the SA2s. It's almost like the map layers are z-index fighting.

[Here's a video](https://www.youtube.com/watch?v=ZGMSpsJKhDE) showing the bug in Chrome on a Mac.

**Comment by LeoSilvaGomes** [comment]
I was reading about the effect of zooming on the map layers and I found some issue on Github that should resolve the problem:
mapbox/mapbox-gl-js#2074
mapbox/mapbox-gl-js#2074 (comment)
I tried to set the opacity to 0.99 to force the layer to appear, but it did not work.

I found another "solution" to the problem, it was to set maxzoom of source: https://stackoverflow.com/questions/54437818/mapbox-gl-geojson-layer-disappears-on-higher-zoom?answertab=active#tab-top
But the layer still disappears when zooming.

My last chance was to analyze the layer. So I realized the style of the map includes some colors layer. I set the background to monochrome on dark mode and tried to analyze the layer again.

To finish I put the dataset on MapBox Studio and tried to find something in the properties that can help me. Then I see that when the map upgrades your data of regions, the layers broke.

https://user-images.githubusercontent.com/61520601/109317094-1cf9ec80-782b-11eb-8da0-092cf8f2c4a9.mp4


https://user-images.githubusercontent.com/61520601/109318164-46674800-782c-11eb-9a3f-bcd4e32b5675.mp4


I guess the issue of this transparency on layers is data inconsistency in our geojson.
